dotnet/src/Extensions/PromptTemplates.Prompty/Experimental.Prompty.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Experimental.Prompty</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Experimental.Prompty</RootNamespace>
```

Comment:
We have model to marking assemblies as experimental rather than using, so follow this pattern instead https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Functions/Functions.Markdown/AssemblyInfo.cs

You can define a new experimental category here https://github.com/microsoft/semantic-kernel/blob/main/dotnet/docs/EXPERIMENTS.md

Rename to `Microsoft.SemanticKernel.Functions.Prompty` and move to the `Functions` folder